### ISSUE TITLE
Table editor: filter panel is buggy

### ISSUE THREAD

**Issue by fifonik** [HeidiSQL/HeidiSQL#394]
## Steps to reproduce this issue
0. Connect to database and open any table
1. Press Ctrl + Alt + F keys combination or select Edit | Filter panel in menu => panel opened and icon is highlighted under Edit menu (OK)
2. When the panel is opened press Ctrl + Alt + F again or select Edit | Filter panel in menu => panel is not closed BUT highlighting from the icon in Edit menu is removed (BUG?)
3. When the panel is opened click at the cross in the panel => panel is closed, but the highlighting from the icon in the Edit menu is NOT removed (BUG?)

## Environment

* **HeidiSQL version:** 9.5.0.5328 64 bit
* **Operating system:** Windows 10 1803

**Comment by fifonik** [comment]
Just a quick question related to the Filter panel.
If I open the panel, type in something to filter columns and then close the panel, should the filtering still be in place or not? 
It is in place when the panel is closed and as a result not all columns are displayed. I've found this very tricky.

Thanks.

**Comment by ansgarbecker** [comment]
Yes, filtering should definitely be disabled when the user cannot see the field for it, otherwise that's very confusing.

**Comment by lukinhaspm** [comment]
Great feature, but I think the CTRL+ALT+F can put by option in menu, like "View" menu, when you can out all this layouts options.
I close in the x, and I came here to found how can I open this.
Thanks

**Comment by ansgarbecker** [comment]
It is in the "Edit" main menu?

**Comment by lukinhaspm** [comment]
Is there, sorry my mistake... I open new request to translate some missing words in Portuguese.

thanks

**Comment by ansgarbecker** [comment]
Please no translation requests here. The translators don't read the tickets here. Instead, you should log in on Transifex and post a message to the Portugesian translator.

**Comment by fifonik** [comment]
I'm confirming that in 5357 there is no columns filtering when the filter is hidden.
Thanks.
